source/grid/grid_out.cc

```diff
@@ -3193,8 +3193,45 @@ namespace internal
           q_projector = new Quadrature<dim> (QProjector<dim>::project_to_all_faces(quadrature));
         }
 
+      std::set<typename Triangulation<dim,spacedim>::active_cell_iterator> interior_cells_to_not_skip;
+      std::vector<Point<spacedim>> point_cache;
       for (; cell!=endc; ++cell)
         {
+          if (!cell->at_boundary() &&
+              interior_cells_to_not_skip.find(cell) == interior_cells_to_not_skip.end())
```

Comment:
This last line can now be written slightly more elegantly as
```
  interior_cells_...skip.count(cell) > 0
```

Reply:
actually, I'm not sure whether I just wrote down the correct condition or its opposite, but that points out a readability problem: variable names with negative names are difficult to understand, and even more so if they have a double negative: "not" and "skip". Can you rename it somehow, and maybe document its purpose where you declare it?